### PR TITLE
Update Virtualization support versions

### PR DESCRIPTION
**Pull request by yhuiYH**

### CODE REVIEW

**Review by amd-jnovotny — APPROVED** [review]
looks good

**Comment by yhuiYH** [comment]
Thx.  Waiting on some other feedback first, before merging.

**Review by lpaoletti — COMMENTED** [review]
Line 2:
Please change the description to:
System requirements for AMD ROCm

File: docs/reference/system-requirements.rst
Comment (lpaoletti) [inline docs/reference/system-requirements.rst]:
```suggestion
    "VMWare", "ESXi 8", "MI210", "Ubuntu 22.04.4 (6.8 [HWE]), |br| SLES 15 SP5 (5.14.21), |br| RHEL 8.10 (5.14.0)"
    "VMWare", "ESXi 7", "MI210", "Ubuntu 22.04.4 (6.8 [HWE]), |br| RHEL 8.10 (5.14.0)"
```

Comment (lpaoletti) [reply docs/reference/system-requirements.rst]: I'm a little surprised we don't have more detail about the ESXi version numbers. 
The website shows more detail - https://knowledge.broadcom.com/external/article/316595/build-numbers-and-versions-of-vmware-esx.html
image

File: docs/reference/system-requirements.rst
Comment (lpaoletti) [inline docs/reference/system-requirements.rst]: Are the column widths supposed to add up to 100?

File: docs/reference/system-requirements.rst
Comment (yhuiYH) [reply docs/reference/system-requirements.rst]: Updated to EXSi 7.0.3 and 8.0.3

File: docs/reference/system-requirements.rst
Comment (yhuiYH) [reply docs/reference/system-requirements.rst]: Fixed

**Pull request merged**
